### ISSUE TITLE
etlas run fails

### ISSUE THREAD

**Issue by qhua948** [rahulmutt/eta-2048#1]
I got this error message when I run `etlas build`
```
Preprocessing executable 'eta2048' for eta2048-0.1.0.0..
Building executable 'eta2048' for eta2048-0.1.0.0..
Running eta2048...
Exception in thread "main" eta.runtime.exception.EtaException: JException java.lang.ClassNotFoundException: org.eta.JavaFXApp extends javafx.application.Application
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:335)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at main.javafx.Main$$Lr10CLlvl6.thunkEnter(Unknown Source)
	at eta.runtime.thunk.CAF.enter(CAF.java:31)
	at eta.runtime.stg.Closure.evaluate(Closure.java:24)
	at main.javafx.Main$$wa.enter(Unknown Source)
	at main.javafx.FRP$playYampa1.enter(Unknown Source)
	at main.Main$main1.enter(Unknown Source)
	at main.Main$DZCmain.enter(Unknown Source)
	at eta.runtime.apply.Function.applyV(Function.java:16)
	at eta.runtime.stg.Closures$EvalLazyIO.enter(Closures.java:96)
	at eta.runtime.stg.Capability.schedule(Capability.java:150)
	at eta.runtime.stg.Capability.scheduleClosure(Capability.java:97)
	at eta.runtime.Runtime.evalLazyIO(Runtime.java:189)
	at eta.runtime.Runtime.main(Runtime.java:182)
	at eta.main.main(Unknown Source)
Caused by: java.lang.ClassNotFoundException: org.eta.JavaFXApp extends javafx.application.Application
	... 20 more
```

I am running Ubuntu 16.04 LTS `Linux 4.10.0-30-generic #34~16.04.1-Ubuntu SMP Wed Aug 2 02:13:56 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux`
I have openjdk8 and openjfx install via `apt-get`

Not sure what is the problem but normal JFX application builds and runs fine on this machine.

**Comment by rahulmutt** [comment]
Thanks for reporting this! The continuous build doesn't run the program, so we didn't catch this before. I have a feeling I know why this is happening - I'll take a look and let you know.

**Comment by Mart-Bogdan** [comment]
@rahulmutt hello, any progres on that?
I'm really excited to run this example! :-) Perhaps some Java src is missing in repo?

it's seem that this is import of Java type:
https://github.com/rahulmutt/eta-2048/blob/0aae40251deb981f439e7f28a33fc6fcafdd6d2d/src/JavaFX/Main.hs#L13
But I just heared about Eta today, so I have no understading :-)

**Comment by rahulmutt** [comment]
Hi @qhua948 and @Mart-Bogdan, this was just fixed on master. The 0.09b2 release will be out in a few weeks with the fix included. If you want to try it out right away, you can download the latest Docker image or install from source. Thanks for your patience!

**Comment by Mart-Bogdan** [comment]
Hi @rahulmutt, you mean new version of etlas?
Good to hear.

**Comment by rahulmutt** [comment]
No, I mean new version of eta :) etlas currently doesn't have the ability to download/install the cutting-edge version (`master` branch of the main repo which is typically kept in a working condition).

**Comment by rahulmutt** [comment]
@qhua948 @Mart-Bogdan I have released 0.0.9b2 which has the fix - you can follow the binary installation instructions and download `etlas` which will automatically download `eta` for you.
